packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,163 @@
+        Opens print terminal
+      </TableCell>
+    </TableRow>
+    <TableRow key="Email">
+      <TableCell key="Utility" headers="utility" stackedTitle="Email Icon">
+        <Button variation="ghost" size="medium">
+          <EmailIcon ariaHidden={false} className="ds-u-margin-right--1" />
+          Email
+        </Button>
+      </TableCell>
+      <TableCell key="Action" headers="action" stackedTitle="Email description.">
+        Opens default desktop email app
+      </TableCell>
+    </TableRow>
+    <TableRow key="RSS">
+      <TableCell key="Utility" headers="utility" stackedTitle="RSS Icon">
+        <Button variation="ghost" size="medium">
+          <RssIcon ariaHidden={false} className="ds-u-margin-right--1" />
+          RSS
+        </Button>
+      </TableCell>
+      <TableCell key="Action" headers="action" stackedTitle="RSS description.">
+        Opens an XML feed in a different tab
+      </TableCell>
+    </TableRow>
+    <TableRow key="Link">
+      <TableCell key="Utility" headers="utility" stackedTitle="Link Icon">
+        <Button variation="ghost" size="medium">
+          <LinkIcon ariaHidden={false} className="ds-u-margin-right--1" />
+          Link
+        </Button>
+      </TableCell>
+      <TableCell key="Action" headers="action" stackedTitle="Link description.">
+        Copies link for current page to clipboard
+      </TableCell>
+    </TableRow>
+    <TableRow key="Download">
+      <TableCell key="Utility" headers="utility" stackedTitle="Download Icon">
+        <Button variation="ghost" size="medium">
+          <DownloadIcon ariaHidden={false} className="ds-u-margin-right--1" />
+          Download
+        </Button>
+      </TableCell>
+      <TableCell key="Action" headers="action" stackedTitle="Download description.">
+        Initiates download
+      </TableCell>
+    </TableRow>
+    <TableRow key="What's New">
+      <TableCell key="Utility" headers="utility" stackedTitle="What's New Icon">
+        <Button variation="ghost" size="medium">
+          <WhatsNewIcon ariaHidden={false} className="ds-u-margin-right--1" />
+          What's New
+        </Button>
+      </TableCell>
+      <TableCell key="Action" headers="action" stackedTitle="What's New description.">
+        Opens sidebar with updates
+      </TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## When to use
+
+- Content on a webpage or a document is something a user would want to be able to refer back to - provide options to print, email, or download.
+- If you would like to give the user the option to:
+  - Open an XML feed in a different tab
+  - Copy a link to a webpage
+  - Open a popup with new updates or new website capabilities
+
+### Usage
+
+- Utility actions are ghost buttons for available actions that do not take priority over primary or secondary call-to-actions (CTAs).
+- Placement should be in the page header and top, right aligned.
```

Comment:
Nit: might be clearer to write "top-right aligned" vs "top, right aligned."

Reply:
I agree, but since this is approved copy we'll keep it as is. Thanks!